go-controller/pkg/util/util.go

```diff
@@ -188,3 +198,30 @@ func HashForOVN(s string) string {
 	return fmt.Sprintf("a%s", hashString)
 
 }
+
+// UpdateIPsSlice will search for values of oldIPs in the slice "s" and update it with newIPs values of same IP family
+func UpdateIPsSlice(s, oldIPs, newIPs []string) []string {
```

Comment:
This would be much simpler if you pass a map old -> new instead of two slices.

Reply:
as we discussed offline this doesnt really help because of the ipfamily checking we still need to iterate through